### DISCUSSION TITLE
Ansible: Failed to connect to the host via scp: /usr/lib/sftp-server: No such file or directory

### DISCUSSION THREAD

**Discussion by akulbe** [vmware/packer-examples-for-vsphere#449] [Help] [answered]
As a follow-on to my previous discussion, I'm still trying to build Almalinux 9.

```
vsphere-iso.linux-almalinux: fatal: [default]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via scp: bash: line 1: /usr/lib/sftp-server: No such file or directory
scp: Connection closed", "unreachable": true}
vsphere-iso.linux-almalinux:
```

I understand this is because of a change made in OpenSSH 9.0, but my control machine that I'm doing the builds from is Fedora 37, and it's running OpenSSH_8.8p1

**Comment by akulbe** [comment]
I tried adding: 

`scp_if_ssh = smart
transfer_method = smart`

to the ansible.cfg but it didn't help

**Comment by tenthirtyam** [comment, marked as answer]
Please note that this project is **only** tested based on the operating systems and versions as noted in the `README.md` - your mileage may vary on alternatives.

Try adding the following to the `ansible/ansible.cfg` file as noted in the `README` for another tested OS.:

```ansible
[ssh_connection]
scp_extra_args = "-O"
```
